src/coordinator/roles.js

```diff
@@ -46,6 +46,102 @@ class Roles {
     this._sdk = sdk;
   }
 
+  /**
+   * Add an application to a role
+   *
+   * API Endpoint: '/applications/:applications_id/roles/:roleId'
+   * Method: POST
+   *
+   * @param {string} organizationId The ID of the organization
+   * @param {string} roleId The UUID formatted ID of the role
+   * @param {string} applicationId The UUID formatted ID of the application
+   *
+   * @returns {Promise}
+   * @fulfill {undefined}
+   * @reject {Error}
+   *
+   * @example
+   * contxtSdk.roles
+   *   .addApplication('4f0e51c6-728b-4892-9863-6d002e61204d', '36b8421a-cc4a-4204-b839-1397374fb16b', '007ca9ee-ece7-4931-9d11-9b4fd97d4d58')
+   *   .then((roleApplication) => console.log(roleApplication))
+   *   .catch((err) => console.log(err));
+   */
+  addApplication(organizationId, roleId, applicationId) {
+    if (!organizationId) {
+      return Promise.reject(
+        new Error('An organizationId is required for adding an application to a role.')
+      );
+    }
+
+    if (!roleId) {
+      return Promise.reject(
+        new Error('A roleId is required for adding an application to a role.')
+      );
+    }
+
+    if (!applicationId) {
+      return Promise.reject(
+        new Error('An applicationId is required for adding an application to a role.')
+      );
+    }
+
+    return this._request
+      .post(`${this._baseUrl}/applications/${applicationId}/roles/${roleId}`)
+      .then((response) => toCamelCase(response));
+  }
+
+  /**
+   * Add a stack to a role
+   *
+   * API Endpoint: '/applications/:applications_id/stacks/:stackId'
+   * Method: POST
+   *
+   * @param {string} organizationId The ID of the organization
+   * @param {string} roleId The UUID formatted ID of the role
+   * @param {string} stackId The UUID formatted ID of the stack
+   * @param {'reader' | 'collaborator' | 'owner'} [accessType='reader'] The level of access for the role
+   *
+   * @returns {Promise}
+   * @fulfill {undefined}
```

Comment:
Same question here about undefined vs. example/formatting to camelcase in the .then

Reply:
thx...done